content/docs/1.1.0/advanced-resources/longhorn-client-python.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Longhorn Client Python
+weight: 5
+---
+
+Currently, you can operate Longhorn using Longhorn UI.
+We are planning to build a dedicated Longhorn CLI in the future.
+While waiting for the CLI, we provide you a Python tool, Longhorn Client Python, for automation/scripting purposes.
+This section servers as a guideline for how to use the Longhorn Client Python to build an automation/scripting tool.
+
+1. Get Longhorn endpoint
+
+   One way to communicate with Longhorn is through `longhorn-frontend` service.
+   If you run your automation/scripting tool inside the same cluster in which Longhorn is installed, connect to the endpoint:
+   ```
+   longhorn_url = 'http://longhorn-frontend.longhorn-system:80/v1'
```

Comment:
`longhorn_url`? I saw it's `LONGHORN_URL` further down.

And is it code? Since you mentioned `kubectl` below, which implies it's bash? But if it's bash, there cannot be space before `=`.

And if it's port `80`, it can be omitted.

Reply:
Sorry for the confusion. It was a mix of bash and Python. I fixed it